docs/sphinx/examples/cpp/providers/quantinuum.cpp

```diff
@@ -19,7 +19,7 @@ struct ghz {
     for (int i = 0; i < 4; i++) {
       x<cudaq::ctrl>(q[i], q[i + 1]);
     }
-    mz(q);
+    auto myResultName = mz(q);
```

Comment:
Note that the language specification leaves the possibility for leaving it as just `mz(q)` in the case of `cudaq::sample`. I'm not sure what the point is in adding the variable name here except to show you can also get the name in the counts output.

Reply:
Yes, this specific change was just to demonstrate that the name will be in the counts output for this test. I ended up adding other tests to demonstrate this, too, so I could remove this particular one if desired.

Reply:
Yea let's remove this one since you added the extra tests.